Expecto/Expect.fs

```diff
@@ -228,7 +229,77 @@ let containsAll (actual : _ seq)
         %s
         Extra elements in `actual`:
         %s"
-              format (formatSet axs) (formatSet exs) (formatSet missing) (formatSet extra)
+              format (formatResult axs) (formatResult exs) (formatResult missing) (formatResult extra)
+  |> Tests.failtest
+
+let inline private except(baseMap: Map<_,int>, exceptMap: Map<_,int>, isExcept: bool) =
+  let getMapValue (map: Map<_, int>) (element: _) = 
+    map.TryFind element |> fun x -> x.Value
+  let getResult found expected = 
+    match isExcept with
+    | true -> (found, expected)
+    | _ -> (expected, found)
+  
+  let differNrOfElement element value =
+    let foundElements = (getMapValue exceptMap element)
+    let missingElements = value - foundElements
+    if missingElements > 0 then getResult foundElements value
```

Comment:
In my opinion by extracting it to a separate variable code is more clear than `if value - foundElements > 0 then ...` or `match value - foundElements > 0 with..`